.env.example

```diff
@@ -17,6 +17,7 @@ VUE_APP_LEGAL_API_URL="https://legal-api-dev.apps.silver.devops.gov.bc.ca"
 VUE_APP_LEGAL_API_VERSION_2="/api/v2"
 VUE_APP_STATUS_API_URL="https://status-api-dev.apps.silver.devops.gov.bc.ca"
 VUE_APP_STATUS_API_VERSION="/api/v1"
+VUE_APP_BUSINESS_EDIT_URL=
```

Comment:
can you add the dev url here so that the pr build picks it up? https://dev.edit.business.bcregistry.gov.bc.ca/